frontend/src/components/ViewActivities/index.js

```diff
@@ -74,24 +75,40 @@ class ViewActivities extends React.Component {
   render() {
     const tripId = this.props.match.params.tripId;    
     if (this.state.error !== undefined) {
-      return <div><ErrorComponents.ErrorGeneral /></div>;
+      return (
+        <div>
+          <Header />
```

Comment:
I realize that the header doesn't need to be imported for the error and loading components, they can be removed (unless you think it looks better with them there).